lib/src/main/java/uk/gov/di/ipv/core/library/service/UserIdentityService.java

```diff
@@ -30,7 +30,7 @@ public class UserIdentityService {
     private static final String PASSPORT_CRI_TYPE = "ukPassport";
     private static final String FRAUD_CRI_TYPE = "fraud";
     private static final String KBV_CRI_TYPE = "kbv";
-    private static final String GPG_45_VALIDITY_PROPERTY_NAME = "validity";
+    private static final String GPG_45_VALIDITY_PROPERTY_NAME = "validityScore";
     private static final String GPG_45_FRAUD_PROPERTY_NAME = "fraud";
     private static final String GPG_45_VERIFICATION_PROPERTY_NAME = "verification";
```

Comment:
Do these also need updating, to identityFraudScore and verificationScore? Had to do similar in https://github.com/alphagov/di-ipv-stubs/pull/99/files

Reply:
Ah yeah good shout, yeah they will as well if we are changing them on the stubs. I'll change that here then and we will need to get this and your PR merged together